pkgs/by-name/ra/rainfrog/package.nix

```
@@ -0,0 +1,33 @@
+{
+  lib,
+  fetchFromGitHub,
+  nix-update-script,
+  rustPlatform,
+}:
+let
+  version = "0.2.4";
+in
+rustPlatform.buildRustPackage {
+  inherit version;
+  pname = "rainfrog";
+
+  src = fetchFromGitHub {
+    owner = "achristmascarl";
+    repo = "rainfrog";
+    rev = "refs/tags/v${version}";
```

```suggestion
    rev = "releases/tag/v${version}";
```

You can then drop the update script as the bot is smart enough to check this path.

---

`refs/tags/*` is correct, since this is a Git ref and not a GitHub URL.

---

Well, nuts. Thank you.